integreat_cms/cms/views/imprint/imprint_sbs_view.py

```diff
@@ -225,5 +262,43 @@ def post(self, request, *args, **kwargs):
                 "imprint_translation_form": imprint_translation_form,
                 "source_imprint_translation": source_imprint_translation,
                 "target_language": target_language,
+                "old_translation_content": old_translation_content,
             },
         )
+
+
+def get_old_source_content(imprint, source_language, target_language):
```

Comment:
This function is just copied + pasted from the page sbs view, right?
Probably we should think about combining these views into more generic ones to remove such duplications. But for now, I guess it's fine... I'll open another issue for this.

Reply:
Do you think AbstractContentModel would be the right place for this function?

Reply:
Would be one possibility, yes. But I opened #1946 for this, so you don't have to change it now if you don't want to.